scripts/verify_replies.sh

```
@@ -0,0 +1,60 @@
+    echo "✅ 返信必要コメントなし"
+    exit 0
+fi
+
+TOTAL_REQUIRED=$(echo "$REQUIRED_REPLIES" | wc -l)
+echo "📊 返信必要数: $TOTAL_REQUIRED件"
+
+# 実際の返信をチェック
+echo "🔎 実際の返信状況確認中..."
+SENT_REPLIES=$(curl -s -H "Authorization: Bearer $GITHUB_TOKEN" \
+    "https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/pulls/$PR_NUMBER/comments" | \
+    jq -r '.[] | select(.user.login == "'$(gh api user -q .login)'") | .in_reply_to_id // empty')
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)